tools/resources/__init__.py

```diff
@@ -148,6 +148,11 @@ def __init__(self, notify, collect_ignores=False):
 
         self._ignoreset = MbedIgnoreSet()
 
+        # make sure mbed-os root is added as include directory
+        script_dir = dirname(abspath(__file__))
+        mbed_os_root_dir = normpath(join(script_dir, '..', '..'))
+        self.add_file_ref(FileType.INC_DIR, mbed_os_root_dir, mbed_os_root_dir)
```

Comment:
Is there a case where not having this breaks the build? I don't believe this should be necessary.